content/python/concepts/strings/terms/isdigit/isdigit.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.isdigit()'
+Description: 'This method under string class that checks if all the elements in the string are digits, decimals also including special cases like compatibility superscript digits.Returns True in the above cases and also requires atleast one character in the string. Returns False otherwise.'
+
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+The `.isdigit()` method under string class that checks if all the elements in the string are digits, decimals also including special cases like compatibility superscript digits.Returns `True` in the above cases and also requires atleast one character in the string. Returns `False` otherwise.
```

Comment:
```suggestion
The **`.isdigit()`** method under the string class checks if all the elements in the string are digits, applicable elements also include special cases like compatibility superscript digits. The method returns `True` in the above cases and also requires at least one character in the string. Returns `False` otherwise.
```